config/initializers/inflections.rb

```diff
@@ -14,3 +14,6 @@
 # ActiveSupport::Inflector.inflections(:en) do |inflect|
 #   inflect.acronym 'RESTful'
 # end
+ActiveSupport::Inflector.inflections(:en) do |inflect|
+  inflect.acronym 'API'
```

Comment:
@hoangmirs How about we use just `Api` for naming in controllers, this way we will not have to override?

Reply:
But `API` is prettier than `Api` 💅 😆

Reply:
Sure its prettier, but I am not sure if its worth using inflectors for this? Actually in fpx project, we went with `Fpx`, instead of `FPX` to avoid this situation. Maybe it can be good to have compass convention for it.

Reply:
The idea is to follow `ConventionOverConfiguration` approach of rails, and not going against pattern as much as possible unless really required.

Reply:
Yeah, I think we should have something on compass for this.
`api, html` should be `API, HTML` as they are acronyms.
Rails also use `API`
<img width="185" alt="Screen Shot 2020-07-21 at 17 29 54" src="https://user-images.githubusercontent.com/7344405/88044070-1fe77d00-cb78-11ea-8748-23146e39d622.png">

Reply:
@hoangmirs Ok, lets go with `API`